src/screens/GroupInfo/index.js

```diff
@@ -182,7 +186,7 @@ const GroupInfo = () => {
         <>
           <Header isCenter onPress={() => navigation.goBack()} title={memberName()} />
           <View style={styles.lineTop} />
-          <ScrollView nestedScrollEnabled={true}>
+          <ScrollView contentContainerStyle={styles.scrollContainer} nestedScrollEnabled={true}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `contentContainerStyle={styles.scrollContainer}` to the `ScrollView` component is a good way to apply styles directly to the inner content container. Just ensure that the styles being applied are necessary and do not conflict with other styles.
```diff
- <ScrollView contentContainerStyle={styles.scrollContainer} nestedScrollEnabled={true}>
+ <ScrollView contentContainerStyle={[styles.scrollContainer, existingStyles]} nestedScrollEnabled={true}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->